duckduckgo/iOS
# Autofill favicon caching change

Task/Issue URL: https://app.asana.com/0/0/1204139098882276/f
Tech Design URL:
CC:

**Description**:
Update to Autofill favicons to also use the fireproof cache, in line with Android



**Steps to test this PR**:
1. Launch the app and search for the `favicons fireproof location` from console (it's usually about the 7th line printed on launch) and go to the folder in Finder and delete all the favicons stored there (or make a note of those that are there)
2. Visit a website with a favicon e.g. netflix.com
3. Login and save the login when prompted
4. Visit the saved login and confirm the favicon is visible on the main logins list screen and also on the details screen for the login
5. Logout and confirm that when logging back in the favicon is shown on the autofill prompt
6. Check in Finder and confirm the favicon has been saved there
7. Tap the fire button in the app and force quit to clear all the cached tab favicons
8. Confirm the favicon is still visible in the logins list screen and the login details screen
9. Delete the Login and confirm in Finder that, after the toast confirming the deletion is gone, that the favicon has been deleted




**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
